docs/_templates/overrides/metpy.plots.ctables.rst

```diff
@@ -0,0 +1,27 @@
+
+
+ctables
+===================
+
+.. automodule:: metpy.plots.ctables
+
+
+
+   .. rubric:: Functions
+
+   .. autosummary::
+      :toctree: ./
+
+      convert_gempak_table
+      read_colortable
+      resource_listdir
+      resource_stream
+
+   .. rubric:: Classes
+
+   .. autosummary::
+      :toctree: ./
+
+      ColortableRegistry
+      Exporter
```

Comment:
`Exporter` comes from elsewhere and we don't need it documented here (or really at all.)

Reply:
Makes sense. I'll remove it